blueocean-dashboard/src/main/resources/jenkins/plugins/blueocean/dashboard/Messages_de.properties

```diff
@@ -0,0 +1,64 @@
+Activity=Aktivit\u00E4ten
+Artifacts=Artefakte
+Author=Autor
+Branch=Ableger
```

Comment:
> However we translated it in spanish I would think we shoud do the same in german

This is absolutely meaningless, it depends on what terms are used in each language.

Compare 'Computer' in German (which is both more common and less colloquial than 'Rechner') to 'ordinateur' in French.